### PR TITLE
fix that checks for exactly one timeseries

### PR DESCRIPTION
**Pull request by xrajesh**
## Description
Fixes e2e test that is running on on hypershift clusters, where  we  can get multiple timeseries .

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: test/e2e/prometheus_client.go [hunk 1]
```diff
@@ -212,15 +212,6 @@ func GetFirstValueFromPromQuery(body []byte) (float64, error) {
 		return 0, err
 	}
```
Comment (syedriko) [inline test/e2e/prometheus_client.go:214]: The block comment doesn't match the body anymore, says "If body contains multiple timeseries
// GetFirstValueFromPromQuery errors."

Comment (xrajesh) [reply test/e2e/prometheus_client.go:213]: updated.

**Comment by xrajesh** [comment]
close this PR in favour of https://github.com/openshift/lightspeed-operator/pull/249

**Pull request closed without merging**